config/clusters/eks_variables.tf

```diff
@@ -14,7 +16,7 @@ variable "eks_default_worker_group_instance_type" {
 }
 
 variable "eks_default_worker_group_asg_min_capacity" {
-  default     = 3
+  default     = 1
```

Comment:
The minimum of one node could be enough to run Prow microservices-only.
It's anyway auto-scaled by the cluster-autoscaler deployed on Kubernetes.